### PR TITLE
Talha/Rbac

### PR DESCRIPTION
**Pull request by talhainvenxion**
RBAC Functuonality

### CODE REVIEW

**Review by schustmi — APPROVED** [review]
Thank you so much for the quick iterations on this, looks good to me!

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-REACT_APP_BASE_API_URL=https://baris.develaws.zenml.io/api/v1
+REACT_APP_BASE_API_URL=https://michael.develaws.zenml.io/api/v1
```
Comment (htahir1) [inline .env:1]: we shouldnt change this right?

Comment (talhainvenxion) [reply .env:1]: Yes, I think we should go with the michael’s one.

Comment (talhainvenxion) [reply .env:1]: @htahir1 Should I merge it?

**Pull request merged**